alpine/Dockerfile

```
@@ -1,4 +1,5 @@
-FROM hmctspublic.azurecr.io/imported/library/node:12-alpine
+ARG NODE_VERSION
```

add a default so it builds with a plain docker build command
```suggestion
ARG NODE_VERSION=16
```